pages/accountLists/[accountListId]/contacts/[[...contactId]].page.tsx

```diff
@@ -37,22 +37,41 @@ const ContactsPage: React.FC = () => {
     setFilterPanelOpen(!filterPanelOpen);
   };
 
-  const setContactFocus = (id?: string, searchTerm?: string) => {
+  const setContactFocus = (id?: string) => {
     const { contactId: _, ...queryWithoutContactId } = query;
     push(
       id
         ? {
             pathname: '/accountLists/[accountListId]/contacts/[contactId]',
-            query: { ...queryWithoutContactId, contactId: id, searchTerm },
+            query: { ...queryWithoutContactId, contactId: id },
           }
         : {
             pathname: '/accountLists/[accountListId]/contacts/',
-            query: { ...queryWithoutContactId, searchTerm },
+            query: { ...queryWithoutContactId },
           },
     );
     setContactDetailsId(id);
   };
 
+  const setSearchTerm = (searchTerm?: string) => {
```

Comment:
@OzzieOrca I think something like this is better than what I had before. Probably can still be improved but it works well for now.